flink-read-write-hive/README.md

```diff
@@ -57,21 +56,28 @@ to run this example.
    $ docker exec -w /root/flink-read-write-hive flink bash -c "export HADOOP_CLASSPATH=\`hadoop classpath\`; python main.py"
    ```
 
-   You should be able to find the following information printed out in the end
-   of the terminal.
+   The program would create and submit two Flink jobs, the second of which runs
+   infinitely as the data stream read from Hive will not be regarded as finished.
+   You may open a second terminal and execute the following command to check the
+   results saved in local file.
+
+   ```
+   cat data/output/.part-*
+   ```
+
+   You should be able to find the following information printed out in the
+   terminal.
 
    ```
-     item_id  price                  timestamp
-   0  item_1  100.0  2022-01-01 00:00:00 +0800
-   1  item_2  200.0  2022-01-01 00:00:00 +0800
-   2  item_3  300.0  2022-01-01 00:00:00 +0800
-   3  item_1  200.0  2022-01-01 00:01:30 +0800
-   4  item_1  300.0  2022-01-01 00:02:30 +0800
-   5  item_1  400.0  2022-01-01 00:03:30 +0800
+   user_1,item_1,1,"2022-01-01 00:00:00",100.0,100.0
```

Comment:
Can you explain why the values are different now?

Reply:
It was a typo that I might have wrongly copied from another example's expected output. I have fixed it and now the values are the same as the previous implementation.